WordPress/Classes/Networking/TaxonomyServiceRemoteXMLRPC.m

```diff
@@ -118,4 +208,35 @@ - (RemotePostTag *)remoteTagFromXMLRPCDictionary:(NSDictionary *)xmlrpcDictionar
     return tag;
 }
 
+- (NSDictionary *)parametersForPaging:(RemoteTaxonomyPaging *)paging
```

Comment:
I'd like to keep these separate as the APIs are just a tad different and any further implementation would need to be treated as such.